package.json

```diff
@@ -29,7 +29,7 @@
     "md5.js": "1.3.4",
     "readable-stream": "~2.0.0",
     "request": "^2.86.0",
-    "underscore": "~1.8.3",
```

Comment:
@XiaoningLiu Do you recall any reason why ~ was used here instead of ^
Am wondering if us making this change will cause any issues

Reply:
@ramya-rao-a , @XiaoningLiu , could we run some integration tests in order to validate this security fix?